kof-operator/internal/controller/suite_test.go

```diff
@@ -65,6 +65,26 @@ func TestControllers(t *testing.T) {
 	RunSpecs(t, "Controller Suite")
 }
 
+var _ = AfterEach(func() {
+	By("Cleanup all objects")
+  objects := []client.Object{
+  	&kcmv1beta1.ClusterDeployment{},
+  	&kofv1beta1.PromxyServerGroup{},
+  	&grafanav1beta1.GrafanaDatasource{},
+  	&corev1.ConfigMap{},
+  	&corev1.Secret{},
+  	&cmv1.Certificate{},
+  	&promv1.PrometheusRule{},
+  }
+  namespaces := []string{defaultNamespace, ReleaseNamespace}
+  for _, obj := range objects {
+  	for _, ns := range namespaces {
+  		err := k8sClient.DeleteAllOf(ctx, obj, client.InNamespace(ns))
+  		Expect(err).To(Succeed())
+  	}
+  }
```

Comment:
Sorry, there was an indentation bug (spaces vs tabs) in my previous suggestion, here is the fixed one:
```suggestion
	By("Cleanup all objects")
	objects := []client.Object{
		&kcmv1beta1.ClusterDeployment{},
		&kofv1beta1.PromxyServerGroup{},
		&grafanav1beta1.GrafanaDatasource{},
		&corev1.ConfigMap{},
		&corev1.Secret{},
		&cmv1.Certificate{},
		&promv1.PrometheusRule{},
	}
	namespaces := []string{defaultNamespace, ReleaseNamespace}
	for _, obj := range objects {
		for _, ns := range namespaces {
			err := k8sClient.DeleteAllOf(ctx, obj, client.InNamespace(ns))
			Expect(err).To(Succeed())
		}
	}
```